### PR TITLE
Node-agent: Added atoms needed for gluster import cluster

### PR DESCRIPTION
**Pull request by nnDarshan**
This patch adds the atoms needed for the gluster import
cluster flow.

Signed-off-by: nnDarshan <darshan.n.2024@gmail.com>

### CODE REVIEW

**Comment by nnDarshan** [comment]
@r0h4n @shtripat @nthomas-redhat Please review

File: etc/tendrl_definitions_node_agent.yaml [hunk 1]
```diff
@@ -40,11 +40,24 @@ namespace.tendrl.node_agent:
           run: tendrl.node_agent.atoms.node.cmd.Cmd
           type: Create
           uuid: dc8fff3a-34d9-4786-9282-55eff6abb6c3
+        check_node_up:
+          enabled: true
+          inputs:
+            mandatory:
+              - fqdn
+	  outputs:
```
Comment (shtripat) [inline etc/tendrl_definitions_node_agent.yaml]: Remove the tabs

File: etc/tendrl_definitions_node_agent.yaml [hunk 2]
```diff
@@ -40,11 +40,24 @@ namespace.tendrl.node_agent:
           run: tendrl.node_agent.atoms.node.cmd.Cmd
           type: Create
           uuid: dc8fff3a-34d9-4786-9282-55eff6abb6c3
+        check_node_up:
```
Comment (shtripat) [inline etc/tendrl_definitions_node_agent.yaml]: Add help field for all the attributes

File: etc/tendrl_definitions_node_agent.yaml [hunk 3]
```diff
@@ -40,11 +40,24 @@ namespace.tendrl.node_agent:
...
+          enabled: true
+          inputs:
+            mandatory:
+              - fqdn
+	  outputs:
+	    - status
+          name: "check whether the node is up"
```
Comment (shtripat) [inline etc/tendrl_definitions_node_agent.yaml]: Lets not use spaces in name field for atoms/flows

File: tendrl/node_agent/gluster_integration/objects/Tendrl_context/atoms/compare.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+class Compare(object):
+    def run(self, **kwargs):
+        return True
```
Comment (shtripat) [inline tendrl/node_agent/gluster_integration/objects/Tendrl_context/atoms/compare.py]: Should we return empty dict as we talked yesterday?

**Pull request closed without merging**